src/module.c

```diff
@@ -496,16 +496,7 @@ NAPI_METHOD(dcn_context_t_dc_set_event_handler) {
   NAPI_RETURN_INT32(1);
 }
 
-NAPI_METHOD(dcn_context_t_dc_unset_event_handler) {
```

Comment:
I moved this to last to keep alphabetical order. We should instead rename the functions so they are close alphabetically as well and then they will be close implementation wise as well.